serverless/knative-client.adoc

```diff
@@ -1,17 +1,18 @@
 [id="knative-client"]
 = Getting started with Knative Client
 include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
 :context: knative-client
 
 toc::[]
 
 Knative Client (`kn`) is the Knative command line interface (CLI).
-The CLI exposes commands for managing your applications, as well as lower level tools to interact with components of {product-title}.
+`kn` exposes commands for managing your applications, as well as lower level tools to interact with components of {product-title}.
 With `kn`, you can create applications and manage {product-title} projects from the terminal.
 
 == Before you begin
 
-Knative Client does not have its own log in mechanism.
+`kn` does not have its own log in mechanism.
```

Comment:
s/own log in/own login (one word when used as an adj.)/
s/To log in to the cluster you/To log in to the cluster, you/